src/blant-output.c

```diff
@@ -329,26 +342,41 @@ Boolean ProcessGraphlet(GRAPH *G, SET *V, unsigned Varray[], const int k, TINY_G
     }
     if(_outputMode & indexOrbits) {
 	assert(TinyGraphDFSConnected(g,0));
+	char buf[BUFSIZ];
 	if(NodeSetSeenRecently(G,Varray,k) ||
 	    (_sampleMethod == SAMPLE_INDEX && !SetIn(_windowRep_allowed_ambig_set, GintOrdinal))) processed=false;
-	else puts(PrintIndexOrbitsEntry(Gint, GintOrdinal, Varray, k, weight, perm));
+	else puts(PrintIndexOrbitsEntry(buf, Gint, GintOrdinal, Varray, k, weight, perm));
     }
     if(_outputMode & communityDetection) {
 	if(_canonNumEdges[GintOrdinal] < _min_edge_count) processed=false;
 	else if(_communityMode == 'o') ProcessNodeOrbitNeighbors(G, Gint, GintOrdinal, Varray, g, k, weight, perm);
 	else if(_communityMode == 'g') ProcessNodeGraphletNeighbors(G, Gint, GintOrdinal, Varray, g, k, weight, perm);
 	else Fatal("unkwown _communityMode %c", _communityMode);
     }
+
+
+ #if 0
```

Comment:
So... I guess I should look at this?

Reply:
I was never the math person, but here's what I noticed with it when I was playing around withit:
Single-threaded, un-commented: passes regression test
Single-threaded, commented: passes regression test
Multithreaded, un-commented: fails regression tests
Multithreaded, commented: passes regression tests

I lost the notes I took on this when making this change, but I can mess around with it again and tell you the results I get.